packages/circuit-ui/components/Avatar/Avatar.tsx

```diff
@@ -55,45 +61,63 @@ const placeholders = {
     '<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 96 96"><path fill="white" d="M48 18c-7.732 0-14 6.268-14 14s6.268 14 14 14 14-6.268 14-14-6.268-14-14-14zM47.9998 88C61.53 88 73.4913 81.2822 80.73 71c-7.2387-10.2822-19.2-17-32.7303-17-13.5302 0-25.4914 6.7178-32.7302 17 7.2388 10.2822 19.2 17 32.7303 17z"/></svg>',
 };
 
-type StyledImageProps = Omit<AvatarProps, 'size' | 'variant'> & {
+type StyledProps = {
   size: AvatarSize;
   variant: AvatarVariant;
 };
 
-const baseStyles = ({ theme, size }: StyledImageProps & StyleProps) => css`
-  display: block;
+const baseStyles = ({ theme, size }: StyledProps & StyleProps) => css`
   width: ${avatarSizes[size]};
   height: ${avatarSizes[size]};
   box-shadow: 0 0 0 ${theme.borderWidth.kilo} rgba(0, 0, 0, 0.1);
-
   background-color: ${theme.colors.n300};
+`;
+
+const imageStyles = () => css`
+  display: block;
   object-fit: cover;
   object-position: center;
 `;
 
-const borderStyles = ({
+const borderRadiusStyles = ({
   theme,
   variant,
   size,
-}: StyledImageProps & StyleProps) => {
-  let styles = css`
-    border-radius: ${theme.borderRadius.kilo};
-  `;
+}: StyledProps & StyleProps) => {
   if (variant === 'identity') {
-    styles = css`
+    return css`
       border-radius: ${theme.borderRadius.circle};
     `;
-  } else if (size === 'giga') {
-    styles = css`
+  }
+
+  if (size === 'giga') {
+    return css`
       border-radius: ${theme.borderRadius.byte};
     `;
   }
-  return styles;
+
+  return css`
+    border-radius: ${theme.borderRadius.kilo};
+  `;
 };
 
-const StyledImage = styled('img', {
+const Image = styled('img', {
+  shouldForwardProp: (prop) => isPropValid(prop),
+})<StyledProps>(baseStyles, borderRadiusStyles, imageStyles);
+
+const initialStyles = ({ theme, size }: StyledProps & StyleProps) => css`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: ${theme.colors.white};
+  font-size: calc(${avatarSizes[size]} / 2);
```

Comment:
Good idea to make these px values to avoid scaling them for users that bump up font sizes (since this is decorative)

Reply:
Did you pick the size based on observations? Because I'm seeing this for instance:

<img width="319" alt="The 'WW' initials touch the edges of the Avatar" src="https://user-images.githubusercontent.com/35560568/194519875-7b2de1f6-8ec3-48ae-984b-e0410673abc5.png">

Maybe we can make them a tad smaller? 🤔

Reply:
Actually a thought: what about using SVG with a `<text>` node? This seems like the perfect use case (text would scale with the svg). But it would be a pretty big change, happy to look at this as a follow-up PR as well

Reply:
I didn't test with different letters 🙈 Good catch! 

An SVG with a `<text>` node is a fantastic idea, I'll give that a shot. Thanks! 🧠

Reply:
Hm, using an SVG means the text scales nicely between the two Avatar sizes. I haven't been able to find a way to make the initials automatically shrink to fit inside the container though. Any hints?

Reply:
Here's what it looks like as an SVG with `lengthAdjust="spacingAndGlyphs"` (JD, WW, II):

![Screenshot 2022-10-07 at 12 24 35](https://user-images.githubusercontent.com/11017722/194532373-2e6465ec-2061-4a90-86a1-ac17fc4e6090.png)

Reply:
Thanks a lot for exploring the idea 🙌